elements/bulbs-video/components/revealed.js

```diff
@@ -269,6 +245,14 @@ export default class Revealed extends React.Component {
     GoogleAnalytics.init(player, videoMeta.gaPrefix);
     Comscore.init(player, global.BULBS_ELEMENTS_COMSCORE_ID, videoMeta.player_options.comscore.metadata);
 
+    if (this.props.onPlay) {
```

Comment:
I know we needed the click tracking elsewhere on the rMVP, but aren't play and pause already tracked by the default implementation?